docker/apm-server/managed/main.go

```diff
@@ -52,10 +52,19 @@ func setupManagedAPM() error {
 
 	// fetch apm package installed to default policy and verify if it is aligned with
 	// expected setup
-	apmPackagePolicies, err := client.getPackagePolicies(fmt.Sprintf("kuery=ingest-package-policies.policy_id:%s", policy.ID))
+	defaultPolicyPackages, err := client.getPackagePolicies(fmt.Sprintf("kuery=ingest-package-policies.policy_id:%s", policy.ID))
 	if err != nil {
 		log.Fatal(err)
 	}
+	var apmPackagePolicies []packagePolicy
+	for _, p := range defaultPolicyPackages {
+		for _, input := range p.Inputs {
+			if input.Type == "apm" {
+				apmPackagePolicies = append(apmPackagePolicies, p)
+				break
```

Comment:
does it matter that this continues through the other default policy packages?

Reply:
Right now multiple APM packages can be set up for one policy, until this is fixed, it makes sense to keep iterating over all of them.

Reply:
https://github.com/elastic/apm-server/issues/4539